src/components/content.tsx

```diff
@@ -19,3 +19,20 @@ export const ContentSection = ({ content }: { content?: string }) => {
     </BasicSection>
   );
 };
+
+export const NarrowContentSection = ({ content }: { content?: string }) => {
+  return (
+    <BasicSection
+      paddingTop={32}
+      paddingLeft={40}
+      paddingRight={40}
+      paddingBottom={0}
+    >
+      <MjmlColumn>
+        <MjmlText>
+          {content && <div dangerouslySetInnerHTML={{ __html: content }} />}
+        </MjmlText>
+      </MjmlColumn>
+    </BasicSection>
+  );
+};
```

Comment:
I suggest adding new prop `paddingBottom`  in <ContentSection /> to control the bottom padding